### ISSUE TITLE
Plaintext password in registration email

### ISSUE THREAD

**Issue by mrc0mmand** [redhatcz/Thesis-management-system#134]
Is there any reason for a password to be sent over an (possibly) insecure channel in its plaintext form? I guess each user should be capable of remembering/saving their password somewhere safe, after the registration process, so there's no need for this security risk in the first place.

**Comment by jcechace** [comment]
I agree completely -- this will be fixed ASAP (will try to do so tomorrow).

**Comment by jcechace** [comment]
@mrc0mmand  can you clarify when is the plaintext password send? AFAIK we send **temporary** passwords in plaintext (e.g. in case somebody else created an account for you). If you perform regular registration, your actual password is never handled in plaintext.

**Comment by mrc0mmand** [comment]
After registration on https://diplomky.redhat.com I received following email:
```
Subject: [Red Hat theses] Please confirm your registration

Your login details:

Your name: Frantisek Sumsal
Your password*: *my_actual_password*

Please confirm your registration by clicking on the following link:
https://diplomky.redhat.com/confirm/xxxxxxxx


------
* Please delete this email after you have confirmed for security purposes.
```

I created this account myself via the registration page.

**Comment by VaclavDedik** [comment]
Fixed by #139
